includes/class-sensei.php

```diff
@@ -1405,6 +1407,41 @@ public function body_class( $classes ) {
 		return $classes;
 	}
 
+	/**
+	 * For course theme, add a body class with the variation.
+	 *
+	 * @param array $classes Body classes.
+	 *
+	 * @intenal
+	 *
+	 * @return array Body classes.
+	 */
+	public function maybe_add_course_theme_variation_class( $classes ) {
```

Comment:
We should probably add this closer to the functionality that uses it, probably in the `Sensei_Course_Theme` class.

Reply:
> We should probably add this closer to the functionality that uses it, maybe in the Sensei_Course_Theme class.

Agreed, just to share why I'm hesitating a bit to do it and see if you feel it can be valid, even though most of our custom CSS are learning mode specific, we also do add styles for non learning mode state too, because Course theme variation styles are applied to the whole site, even out of learning mode. For example, here we have added some styles for quiz questions https://github.com/Automattic/themes/blob/dafe8da4ade2a571ff068487a7f1214c7aac3158/course/styles/gold.json#L308, these styles are meant to be applied on questions even out of learning mode. But `Sensei_Course_Theme` class is for learning mode only. WDYT?

Reply:
Sure, makes sense and not a big deal. 👍🏻